doc/source/user_guide/io.rst

```diff
@@ -2714,156 +2714,34 @@ succeeds, the function will return*.
 Writing to HTML files
 ''''''''''''''''''''''
 
-``DataFrame`` objects have an instance method ``to_html`` which renders the
-contents of the ``DataFrame`` as an HTML table. The function arguments are as
-in the method ``to_string`` described above.
-
 .. note::
 
-   Not all of the possible options for ``DataFrame.to_html`` are shown here for
-   brevity's sake. See :func:`~pandas.core.frame.DataFrame.to_html` for the
-   full set of options.
+   DataFrame *and* Styler objects currently have a ``to_html`` method. We recommend
+   using the `Styler.to_html() <../reference/api/pandas.io.formats.style.Styler.to_html.rst>`__ method
```

Comment:
Can you use the sphinx cross-reference syntax instead of the path-based link? (``` :meth:`Styler.to_html() <pandas.io.formats.style.Styler.to_html>` ``` for the long format and controlling which text is changes, but I think in this case you can also use ``` :meth:`.Styler.to_html` ``` to get exactly the same)

(and the same for similar cases below)